simple-weather/simple-weather/Controller/ViewController.swift

```diff
@@ -58,3 +65,16 @@ class ViewController: UIViewController, CLLocationManagerDelegate {
         }
     }
 }
+
+extension ViewController: UITableViewDelegate, UITableViewDataSource {
```

Comment:
And remove conformance as well `UITableViewDelegate `

Reply:
But when I delete delegate function from override function the simulator is not showing any cell values. I was struggling with that already yesterday, that's why I added delegate to override function. What should I change to see data in simulator?


<img width="1440" alt="Screenshot 2019-04-26 at 12 25 08" src="https://user-images.githubusercontent.com/41584015/56798014-af3f9c80-681e-11e9-94ed-d44d9a4291dd.png">

Reply:
I got your idea now! I that that I need delete all that is in delegate function. Now that I adde dataSource it works. Thanks!